css/src/components/horizontalNav.css

```diff
@@ -16,6 +16,11 @@
 
 .HorizontalNav-menu--disabled {
   pointer-events: none;
+  color: var(--inverse-lightest);
+}
+
+.HorizontalNav-menu--default {
+  color: var(--inverse);
```

Comment:
We are using the same colours for MenuText as well. Please create a common function to get colours for both MenuWrapper and MenuText and reuse it.